src/main/java/kr/motd/maven/os/Detector.java

```diff
@@ -54,6 +69,11 @@ protected void detect(Properties props) throws DetectionException {
                 throw new DetectionException("unknown os.arch: " + osArch);
             }
         }
+
+        for (String like : getOsLike(detectedName)) {
```

Comment:
I have no problem just doing this when detectedName == "linux".  If we're going to go that route, then I won't bother adding `detectedName` to the list, and non-linux platforms just won't have any `like` properties available.